### ISSUE TITLE
Model still detected as 'Camunda Platform' although modeler:executionPlatform="Camunda Cloud"

### ISSUE THREAD

**Issue by falko** [camunda/camunda-modeler#2279]
__Describe the Bug__

The attached model is still detected as 'Camunda Platform' although it explicitely defines modeler:executionPlatform="Camunda Cloud"

__Steps to Reproduce__

1. Open this file [customer_onboarding_en.bpmn.txt](https://github.com/camunda/camunda-modeler/files/6513575/customer_onboarding_en.bpmn.txt)
2. Status bar shows 'Camunda Platform'

__Expected Behavior__

Status bar shows 'Camunda Cloud'

__Environment__

- OS: Kubuntu 20.04.2 LTS
- Camunda Modeler Version: 4.9.0-nightly.20210518
- Execution Platform: Camunda Cloud
- Installed plug-ins: Cloud Connect

**Comment by falko** [comment]
I introduced the execution platform attribute via `sed -i 's#xmlns:camunda="http://camunda.org/schema/1.0/bpmn"#xmlns:camunda="http://camunda.org/schema/1.0/bpmn" xmlns:zeebe="http://camunda.org/schema/zeebe/1.0" xmlns:modeler="http://camunda.org/schema/modeler/1.0" modeler:executionPlatform="Camunda Cloud" modeler:executionPlatformVersion="1.0.0"#g' *.bpmn` and that worked fine for other files.

**Comment by falko** [comment]
Okay, now it worked for me after I closed the file and opened it again using the open file dialogue. It seems like the reload after the file got modified externally as well as closing the file and then 'reopen last file' do not check for the platform again.

**Comment by pinussilvestrus** [comment]
Yes, we are checking for the platform only on file opening (when we retrieve the correct tab type for the diagram). Makes sense to maybe do the checks way more often.

**Comment by falko** [comment]
Yes, please ;-)

**Comment by falko** [comment]
Well, you might as well fix the root cause and have an easy way to open Zeebe Modeler and Cawemo files as Camunda Cloud files or have an easy way to switch as discussed in #2283

**Comment by nikku** [comment]
@pinussilvestrus How complicated would https://github.com/camunda/camunda-modeler/issues/2279#issuecomment-845042751 be?

**Comment by pinussilvestrus** [comment]
It's not trivial. Our `checkFileChanged` handler is currently assuming the [type of the tab did not change](https://github.com/camunda/camunda-modeler/blob/develop/client/src/app/App.js#L256). We would have to break this up and recreate the tab if necessary.

**Comment by nikku** [comment]
We'll address the root cause of this issue by removing the `+` button, at least temporarily.

**Comment by falko** [comment]
The `+` button was not the root cause of this issue.

**Comment by berndruecker** [comment]
Could we rename this issue to "File type should be re-checked upon reload" to sort out the different problems?

Even the "Reopen last file" does not do a fresh check. This is pretty confusing in this context.
I also face this problem when applying my little cloud -> platform migration plugin.

**Comment by nikku** [comment]
I think this will be relevant as migration to Camunda 8 becomes more of a thing. Linked to relevant places, potential spring cleaning topic.
